Source: dleverton
Upsream: no
Reason: suppress stupid "QA" warning that breaks test suites such as libtool's

diff --git a/libsandbox/wrapper-funcs/execve.c b/libsandbox/wrapper-funcs/execve.c
index cc8f7a8..0eb8fa7 100644
--- a/libsandbox/wrapper-funcs/execve.c
+++ b/libsandbox/wrapper-funcs/execve.c
@@ -56,7 +56,6 @@ void check_exec(const char *filename)
 		PARSE_ELF(32);
 	else
 		PARSE_ELF(64);
-	SB_EWARN("QA: Static ELF", " %s\n", filename);
  done:
 
  out_mmap:
